### ISSUE TITLE
Perfect score animation acting buggy

### ISSUE THREAD

**Issue by tannerdolby** [GoogleChrome/lighthouse#12563]
Being a longtime Lighthouse user, I have come to really enjoy the perfect score animation. The confetti animation that follows and shoots confetti off in realistic *poof* is great. I've done this sort of "blast something into the air" animation with GSAP and Physics2D before, not sure how the animation is being implemented here (Pure CSS, some JS, idk) in Lighthouse without having a deeper look at the source.

That all being said, the animation looks great (as expected). But, I've noticed some unexpected behavior for the confetti animation. When you scroll the page down inside the Lighthouse audit (after results are shown) assuming a perfect score was received, the confetti animation starts acting a bit odd and the animation becomes a bit more choppy with the confetti becoming slightly visible as its blurs across the screen to new positions before "firing" more confetti. 

When does this happen:

1. Intermittently when the Lighthouse audit results page isn't scrolled and a perfect score is received.
2. When a lighthouse audit completes and you receive a perfect score, then scroll down inside the audit window and back up to the animation.

Note: In chrome Version 90.0.4430.212 (Official Build) (x86_64) on Mac OS High Sierra

I'm sure most people won't notice this slight sporadic behavior, but maybe some will. Below is a short video highlighting how the confetti begin to become visible as they whizz across the screen and move to different positions before firing off confetti. 

https://user-images.githubusercontent.com/48612525/119689717-efa6e580-bdfd-11eb-9c5f-e84b814bef07.mov

I'm going to have a look in the repo and see if I can `git grep -n ""` anything to see where the animation is being implemented. 

My gut thought for a quick fix is to ensure that the opacity or 'transparency' is 0 when the "confetti launcher" is moving to a new position on the screen and launching confetti. It seems that there is a 1-2ms gap (probably less) where the opacity of the confetti is 1 and they are completely visible as we watch them zip across the screen in a sort of abrupt manner.

**Comment by tannerdolby** [comment]
Update: The bug doesn't seem to be reproduced in Chrome on Windows, also checked Firefox, Opera, Edge, no bugs. Unless someone can reproduce the erratic behavior of the confetti moving positions as shown in the video above using a Windows OS in any browser, it seems this issue is only concerning MacOS in Chrome.

**Comment by adamraine** [comment]
I've experienced this too. Since filing this bug, confetti has stopped appearing altogether for me.

**Comment by tannerdolby** [comment]
@adamraine Ok, glad I wasn't the only one noticing it. Yeah I can also confirm confetti has completely stopped appearing for perfect scores now. Just gave it a run on https://metagendocs.netlify.app/ which receives a perfect score but without any animation.

I wonder if the animation has been removed completely in source or something is broken? I got up-to-date with the latest `master` and will give it a look. This small part of Lighthouse was something I really enjoyed. When first opening this issue, I created a [CodePen](https://codepen.io/tannerdolby/pen/WNjRXwg) to try pinpointing where in the animation the buggy behavior occurred and "sort of" made progress by utilizing the "Animations" tab and walking through the animation to see exactly at what time in the animation (keyframes in this case) the position shifting by varying the margins lags and creates the issue of seeing the visible element whizz from one position to the next when it should be transparent while moving to new positions.
